stdlib/public/core/Integers.swift.gyb

```diff
@@ -3590,10 +3590,12 @@ ${assignmentOperatorComment(x.operator, True)}
   /// to find an absolute value. In addition, because `abs(_:)` always returns
   /// a value of the same type, even in a generic context, using the function
   /// instead of the `magnitude` property is encouraged.
-  @_transparent
   public var magnitude: U${Self} {
-    let base = U${Self}(_value)
-    return self < (0 as ${Self}) ? ~base + 1 : base
+    @inline(__always)
+    get {
```

Comment:
This seems unrelated?

Reply:
This is needed because constant propagation now looks at both branches, so as a way to work around that behavior @ravikandhadai and @moiseev told me that it would be appropriate to switch from `@_transparent` to `@inlineable`.

Reply:
It is, unfortunately. `@_transparent` here causes the compile-time overflow on a branch that would not be taken at runtime.

Reply:
Just to clarify. The changes to how initializers are resolved, as a side-effect, makes constant propagation more precise as it can now propagate through more initializers. While in general it is a good thing, in this specific case, it resulted in a false positive due to mandatory inlining.

Reply:
It is very much possible (and probably reasonable) to try the `&+` approach in a separate PR once this lands.

Reply:
@moiseev @xwu there is one other issue with "magnitude" (besides the +) that could trigger static errors if it is made @_transparent. It is passing a potentially negative (builtin Int) into a UInt : `let base = U${Self}(_value)`. This could trigger a static error on negative values like (-1).magnitude etc.  So perhaps keeping it @inline is better.

Reply:
No, `_value` is [the same type](https://github.com/xedin/swift/blob/29bbc99780265b6800f917f50f24f6818e9ae8e9/stdlib/public/core/Integers.swift.gyb#L3710) regardless of whether it's `Int` or `UInt`, and that initializer simply assigns `self._value = other._value`.